airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/destination/dest_state_lifecycle_manager/DestStreamStateLifecycleManager.java

```diff
@@ -62,16 +66,36 @@ public Queue<AirbyteMessage> listFlushed() {
     return listStatesInOrder(streamToLastFlushedState);
   }
 
+  /*
+   * During the process of migration to destination checkpointing, this method should no longer be in
```

Comment:
Does this comment imply this method is deprecated?

Reply:
Yes, this should be marked as `Deprecated`, thanks for the catch. It appears during the rebase I _mistakenly_ deleted the `@Deprecated` tag